stl/inc/yvals_core.h

```diff
@@ -267,6 +267,7 @@
 // P1682R3 to_underlying() For Enumerations
 // P1951R1 Default Template Arguments For pair's Forwarding Constructor
 // P1989R2 Range Constructor For string_view
+// P2136R3 invoke_r()

```

Comment:
_HUGE_ nit: the paper title does not seem to have the parenthesis

Reply:
Good catch - this is intentional, I added the parentheses to the "cleaned-up" title in #1978 to be consistent with the other cleaned-up titles.